android/libraries/rib-android-test/src/test/java/com/uber/rib/core/RibActivityTest.java

```diff
@@ -202,7 +202,7 @@ public void bind_afterDestroy_shouldError() {
         Robolectric.buildActivity(EmptyActivity.class);
     EmptyActivity activity = activityController.setup().pause().stop().destroy().get();
     AndroidRecordingRx2Observer<Object> o = new AndroidRecordingRx2Observer<>();
-    Observable.just(new Object()).to(new ObservableScoper<>(activity)).subscribe(o);
+    Observable.just(new Object()).as(AutoDispose.autoDisposable(activity)).subscribe(o);
```

Comment:
Maybe use static import for ausdisposable to demo best practice (keeping readable)?  Up to you

Reply:
9cb9f90